client/client.go

```diff
@@ -327,6 +327,26 @@ func (w *WrapperV2) CreateKeyPair(clusterID string, addKeyPairRequest *models.V4
 	return response, nil
 }
 
+// GetKeyPairs calls the API to fetch key pairs.
+func (w *WrapperV2) GetKeyPairs(clusterID string, p *AuxiliaryParams) (*key_pairs.GetKeyPairsOK, error) {
+	if w == nil {
+		return nil, microerror.Mask(clientV2NotInitializedError)
+	}
+
+	params := key_pairs.NewGetKeyPairsParams().WithClusterID(clusterID)
+	setParams(p, w, params)
+	if w.conf.AuthHeader != "" {
+		params.SetAuthorization(w.conf.AuthHeader)
+	}
```

Comment:
This one here can also by abstracted by interface embedding.

Example:

```
type paramSetterAuthorization interface {
	paramSetter

	SetAuthorization(string)
}

func setParamsAuthorization(p *AuxiliaryParams, w *WrapperV2, params paramSetterAuthorization) {
	if w != nil && w.conf != nil {
		if w.conf.AuthHeader != "" {
			params.SetAuthorization(w.conf.AuthHeader)
		}
	}

	setParams(p, w, params)
}
```

Then you call `setParamsAuthorization(p, w, params)` instead of `setParams(p, w, params)`.